oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/dsl/DSL.java

```diff
@@ -37,6 +38,9 @@
     public static Expression parse(final String expression) {
         CompilerConfiguration cc = new CompilerConfiguration();
         cc.setScriptBaseClass(DelegatingScript.class.getName());
+        ImportCustomizer icz = new ImportCustomizer();
+        icz.addImport("K8sRetagType", "org.apache.skywalking.oap.meter.analyzer.dsl.SampleFamily.K8sRetagType");
```

Comment:
```suggestion
        icz.addImport("K8sRetagType", K8sRetagType.class.getName());
```
This should be safer for potential refactor.

Reply:
done